.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     resource_class: xlarge
     working_directory: /go/src/github.com/networkservicemesh/networkservicemesh
     docker:
-      - image: circleci/golang:1.12-stretch
```

Comment:
Maybe we could use some env variable to put version inside?

Reply:
On other hand, this values here show us a version used, and if will be some env variable in CI we will not know it value.